precedence.js

```diff
@@ -0,0 +1,59 @@
+(function() {
+    "use strict";
+
+    function Precedence() {
+        this.numbers = [];
+        this.operations = [];
+        this.turnOn = false;
+        this.finalResult = 0;
+    }
+
+    Precedence.prototype.returnValue = function() {
+        this.result = Number(this.numbers[0]);
+
+        for (var i = 0; i < this.numbers.length; i++) {
+            if (this.operations[i] === 'x') {
```

Comment:
operations[i] can be just one symbol. But your approach checks for everything. For example if it has value 'x' than you would check also for '/', '-', etc. In cases like this one, always use if else:
```
if (a) { ... }
else if (b) {...}
else {...}
```